### PR TITLE
boylsojh - code review

### PR DESCRIPTION
**Pull request by joeboylson**
The Java code all looked pretty good to me and I really didn't have anything to say.
I know the UI is temporary but I thought I'd just add a few changes and best practice (feel free to just ignore all these).

### CODE REVIEW

**Comment by joeboylson** [comment]
# ASSIGNMENT INFO


---

### Analysis of the program.
For where we are in the class, I think this program has the sufficient information and UI. It runs just fine and has a few stubbed out pages.

### Was the program available in UC Github on time?
Yes.

### Is the program documented/commented well enough for you to understand?
Yes, the comments were definitely desciptive.

### Does the program compile?
Yes.

File: src/main/resources/static/style.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 table {
-	width: 1400px;
+	width: 100%;
```
Comment (discospiff) [inline src/main/resources/static/style.css:5]: Good idea... a pixel value, like 1400px will not look good on most devices.  Percentage is much more flexible.

File: src/main/resources/static/style.css [hunk 2]
```diff
@@ -12,4 +12,23 @@ th {
...
+.height-100 {
+	height: 100px;
+}
+
+.height-50 {
+	height: 50px;
+}
+
+.link-button {
+	background-color: lightgray; /* or whatever  */
```
Comment (discospiff) [inline src/main/resources/static/style.css:30]: Good descriptions, but I'd put them in the pull request.

File: src/main/resources/templates/athlete.html [hunk 3]
```diff
@@ -1,41 +1,42 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<meta charset="UTF-8"></meta>
-<link rel="stylesheet" type="text/css" media="screen" href="style.css"/>
-<title>Athlete Workout Page</title>
+	<meta charset="UTF-8">
```
Comment (discospiff) [inline src/main/resources/templates/athlete.html:5]: Much easier to read with indentations!

File: src/main/resources/templates/athlete.html [hunk 4]
```diff
@@ -1,41 +1,42 @@
...
+			<th>Tuesday</th>
+			<th>Wednesday</th>
+			<th>Thursday</th>
+			<th>Friday</th>
+			<th>Saturday</th>
+			<th>Sunday</th>
+		</tr>
+		<tr>
+			<th class="height-600"></th>
+			<th></th>
```
Comment (discospiff) [inline src/main/resources/templates/athlete.html:32]: These probably need to be td tags.  th is usually only used on the very first row, for the table header.

File: src/main/resources/templates/editAthlete.html [hunk 5]
```diff
@@ -6,17 +6,12 @@
...
-	</a>
-	</br>
-	</br>
-	<a href="index">
-		<button>Home</button>
-	</a>
-	</br>
-	</br>
+
+	<a class='link-button' href="athleteList">Back</a>
```
Comment (discospiff) [inline src/main/resources/templates/editAthlete.html:12]: Good suggestion... and it takes a keen eye to see what is going on here.

- Many developers use multiple br tags, where css margin or padding properties are better suited.

- So, good idea to remove br, and replace with css, which can vary the margin based on the user's viewing device.

File: src/main/resources/templates/editAthlete.html [hunk 6]
```diff
@@ -29,7 +24,7 @@ <h1>Edit Athlete Schedule Page</h1>
 		<th>Sunday</th>
 	</tr>
 	<tr>
-		<th height="600"></th>
+		<th class="height-600"></th>
```
Comment (discospiff) [inline src/main/resources/templates/editAthlete.html:27]: Great again.  Move style external.  More flexible, less redundant.

**Comment by discospiff** [comment]
A really nice review.  Several changes here that reduce technical debt.  I recommend merging these changes.

**Pull request merged**